### PR TITLE
Update define-command-argument-container.md

### PR DESCRIPTION
**Pull request by city2011**
Use uppercase letter in note and add args corresponding description to help people search who familiar with CMD ENTRYPOINT

### CODE REVIEW

File: content/en/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ If you define args, but do not define a command, the default command is used
 with your new arguments.
 
 {{< note >}}
-The `command` field corresponds to `entrypoint` in some container runtimes.
+The `command` field corresponds to `ENTRYPOINT`, `args` field corresponds to `CMD` in some container runtimes.
```
Comment (salaxander) [inline content/en/docs/tasks/inject-data-application/define-command-argument-container.md:39]: I would suggest a small grammatical change

```suggestion
The `command` field corresponds to `ENTRYPOINT`, and the `args` field corresponds to `CMD` when `ENTRYPOINT` is used.
```

Comment (city2011) [reply content/en/docs/tasks/inject-data-application/define-command-argument-container.md]: I remain [and the] to fix grammatical error. and use [in some container runtimes] before.

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!
Congrats on your first PR 🎉
/lgtm

[Preview Page](https://deploy-preview-45900--kubernetes-io-main-staging.netlify.app/docs/tasks/inject-data-application/define-command-argument-container/)

**Pull request merged**